ci/gpu/build.sh

```diff
@@ -61,8 +61,9 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
 # https://docs.rapids.ai/maintainers/depmgmt/
 # gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
 # gpuci_conda_retry install -y "your-pkg=1.0.0"
-gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env
+gpuci_conda_retry remove --force rapids-build-env rapids-notebook-env faiss
 gpuci_conda_retry install "libgcc-ng=9.3.0" "libstdcxx-ng=9.3.0" "libgfortran-ng=9.3.0"
+gpuci_conda_retry install "conda-forge::faiss=1.6.3" "conda-forge::faiss-proc=*=cuda"
```

Comment:
there's 1.6.5 on conda-forge as well, any reason we can't upgrade?

Reply:
not entirely sure, at some point there was some file re-organization and we would need some minor adjustments

Reply:
I can also remove our version of faiss if we can verify that conda-forge has a version for CUDA 11.0

Reply:
Yeah it was added in PR ( https://github.com/conda-forge/faiss-split-feedstock/pull/13 )

Reply:
probably the best path after everything passes with the conda-forge package

Reply:
Looks like this fixed the issues 🎉